src/scripts/main.js

```diff
@@ -1,7 +1,94 @@
 'use strict';
 
-// Uncomment the next lines to use your game instance in the browser
-// const Game = require('../modules/Game.class');
-// const game = new Game();
+const Game = require('../modules/Game.class');
+const game = new Game();
 
-// Write your code here
+const container = document.querySelector('.container');
+const gameField = document.querySelector('.game-field');
+const button = container.querySelector('.button');
+const gameScore = container.querySelector('.game-score');
+
+const fieldRows = [...gameField.querySelectorAll('.field-row')];
+const fieldCells = fieldRows.map((row) => [...row.children]);
+
+const messages = {
+  idle: container.querySelector('.message-start'),
+  lose: container.querySelector('.message-lose'),
+  win: container.querySelector('.message-win'),
+};
+
+function fillGameField(state) {
+  state.forEach((row, i) => {
+    row.forEach((cell, j) => {
+      const currentCell = fieldCells[i][j];
+
+      currentCell.className = !cell
+        ? 'field-cell'
+        : `field-cell field-cell--${cell}`;
+      currentCell.innerHTML = !cell ? '' : cell;
```

Comment:
It's good practice to avoid repeating conditions in ternary operators. You can simplify the assignment by using a template literal and including the condition once.